firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -60,6 +60,32 @@ class DictionaryEntryDetailSerializer(serializers.HyperlinkedModelSerializer):
     )
     categories = CategorySerializer(many=True)
     site = SiteSummarySerializer()
+    base_characters = serializers.SerializerMethodField()
+    words_in_phrase = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_base_characters(entry):
+        if "⚑" in entry.custom_order:
+            return "Due to this entry containing unknown characters, it cannot be used in games."
+        else:
+            alphabet = Alphabet.objects.filter(site=entry.site).first()
+            cs = alphabet.sorter
+            character_list = cs.word_as_chars(entry.title)
+            for i, char in enumerate(character_list):
+                character_list[i] = alphabet.presort_transducer(char).output_string
+
+            return character_list
+
+    @staticmethod
+    def get_words_in_phrase(entry):
+        if entry.type != "PHRASE":
+            return "This entry is not a phrase."
```

Comment:
Please don't return messages for humans like this. We can either return a value for computers, or raise an error with a message for humans. For this case, how about just returning the title since there is only one word in a "word"?